zulipterminal/config/keys.py

```diff
@@ -471,6 +471,49 @@ def commands_for_random_tips() -> List[KeyBinding]:
     ]
 
 
+def override_keybindings(
+    custom_keybindings: Dict[str, str], existing_keybindings: Dict[str, KeyBinding]
+) -> None:
+    reverse_key_map = {
+        key: cmd
+        for cmd, binding in existing_keybindings.items()
+        for key in binding["keys"]
+    }
+    requested_changes = {}
+    conflicts = {}
+
+    # Collect requested changes and detect conflicts
+    for command, new_key in custom_keybindings.items():
+        if command not in existing_keybindings:
+            raise InvalidCommand(f"Invalid command {command} in custom keybindings")
```

Comment:
Errors from this method should be handled and cleanly expressed to the user in run.py, not just left to escape unhandled.